pyscf/lo/reg_emb.py

```diff
@@ -192,10 +198,26 @@ def get_virt_frozen(self, threshold=None):
             threshold = self.threshold
         return self.virt_loc_mos[:, numpy.abs(self.virt_frag_mo_ovlp) < threshold]
 
-    def kernel(self, threshold=None):
+    def print_thresholds(self, percentages=None):
+        frag_mo_ovlp = numpy.sort(numpy.concatenate([self.occ_frag_mo_ovlp, self.virt_frag_mo_ovlp]))[::-1]
+        self.log.info("Analyzing orbital inclusion with respect to threshold")
+        self.log.info("{}{}{}".format("-"*40, "-"*10, "-"*30))
+        self.log.info("{:>40}          {:>30}".format("Pct of full space", "Threshold"))
+        self.log.info("{}{}{}".format("-"*40, " "*10, "-"*30))
+        if (percentages == None):
+            percentages = numpy.arange(5,100,5) / 100.0
+            orbital_nums = (len(frag_mo_ovlp) * percentages).astype(int)
+        for percent,idx in zip(percentages, orbital_nums):
+            if idx == 0 or idx == len(frag_mo_ovlp):
+                thresh_to_use = frag_mo_ovlp[idx]
+            else:
+                thresh_to_use = (frag_mo_ovlp[idx+1] + frag_mo_ovlp[idx]) / 2.0
+            self.log.info("{:>39.6f}           {:>30.16f}".format(percent, thresh_to_use))
+        self.log.info("{}{}{}".format("-"*40, "-"*10, "-"*30))
+
+    def partition_orbitals(self, threshold=None):
         if(threshold == None):
             threshold = self.threshold
-        self.rotate_orbs()
```

Comment:
is this being removed so we call rotate orbitals followed by partition orbitals without a kernel function?